eclipse-scout-core/src/table/Table.ts

```diff
@@ -4978,9 +4987,7 @@ export class Table extends Widget implements TableModel, Filterable<TableRow> {
    */
   setLimitedResultTableStatus(limitedResult: boolean) {
     if (!limitedResult) {
-      if (this.tableStatus instanceof LimitedResultTableStatus) {
```

Comment:
Ask mvi why this was done and/or test it. Is it possible that an existing tableStatus is now removed? tableStatus may be set by ui server

Reply:
If there was an error status because of a loading error, it did not vanish anymore even if data could be loaded later. Custom error status would be replaced, yes, but the same happens in Java -> not supported yet